exporters/trace/stackdriver/src/test/java/io/opencensus/exporter/trace/stackdriver/StackdriverV2ExporterHandlerProtoTest.java

```diff
@@ -21,6 +21,7 @@
 import static io.opencensus.contrib.monitoredresource.util.ResourceType.GCP_GCE_INSTANCE;
 import static io.opencensus.contrib.monitoredresource.util.ResourceType.GCP_GKE_CONTAINER;
 import static io.opencensus.exporter.trace.stackdriver.StackdriverV2ExporterHandler.createResourceLabelKey;
+import static io.opencensus.exporter.trace.stackdriver.StackdriverV2ExporterHandler.getResourceLabels;
```

Comment:
Suggestion: I think the static import makes the test harder to read, because it's hard to see whether this is a non-test method or a test helper method without looking at the imports.